pep-0297.txt

```diff
@@ -5,112 +5,125 @@ Last-Modified: $Date$
+========
 
-    This PEP proposes strategies to allow the Python standard library
-    to be upgraded in parts without having to reinstall the complete
-    distribution or having to wait for a new patch level release.
+This PEP proposes strategies to allow the Python standard library
+to be upgraded in parts without having to reinstall the complete
+distribution or having to wait for a new patch level release.
 
 
 Problem
+=======
+
+Python currently does not allow overriding modules or packages in
+the standard library per default. Even though this is possible by
+defining a ``PYTHONPATH`` environment variable (the paths defined in
+this variable are prepended to the Python standard library path),
+there is no standard way of achieving this without changing the
+configuration.
 
-    Python currently does not allow overriding modules or packages in
-    the standard library per default. Even though this is possible by
-    defining a PYTHONPATH environment variable (the paths defined in
-    this variable are prepended to the Python standard library path),
-    there is no standard way of achieving this without changing the
-    configuration.
+Since Python's standard library is starting to host packages which
+are also available separately, e.g. the distutils, email and PyXML
+packages, which can also be installed independently of the Python
+distribution, it is desirable to have an option to upgrade these
+packages without having to wait for a new patch level release of
+the Python interpreter to bring along the changes.
 
-    Since Python's standard library is starting to host packages which
-    are also available separately, e.g. the distutils, email and PyXML
-    packages, which can also be installed independently of the Python
-    distribution, it is desirable to have an option to upgrade these
-    packages without having to wait for a new patch level release of
-    the Python interpreter to bring along the changes.
+On some occasions, it may also be desirable to update modules of
+the standard library without going through the whole Python release
+cycle, e.g. in order to provide hot-fixes for security problems.
 
-    On some occasions, it may also be desirable to update modules of
-    the standard library without going through the whole Python release
-    cycle, e.g. in order to provide hot-fixes for security problems.
 
 Proposed Solutions
+==================
 
-    This PEP proposes two different but not necessarily conflicting
-    solutions:
+This PEP proposes two different but not necessarily conflicting
+solutions:
 
-    1. Adding a new standard search path to sys.path:
-       $stdlibpath/system-packages just before the $stdlibpath
-       entry. This complements the already existing entry for site
-       add-ons $stdlibpath/site-packages which is appended to the
-       sys.path at interpreter startup time.
+1. Adding a new standard search path to sys.path:
+   $stdlibpath/system-packages just before the $stdlibpath
```

Comment:
\`\`$stdlibpath/system-packages\`\`

\`\`$stdlibpath\`\`

(there are multiple instances that needs to be formatted)